packages/cdk8s/src/_loadurl.js

```diff
@@ -17,13 +17,11 @@ if (!url) {
 
 const purl = parse(url);
 
-if (!purl.protocol) {
-  if (!purl.pathname) {
-    throw new Error(`unable to parse pathname from file url: ${url}`);
+try {
+  if (fs.lstatSync(url).isFile()) {
+    fs.createReadStream(url).pipe(process.stdout);
   }
-
-  fs.createReadStream(purl.pathname).pipe(process.stdout);
-} else {
+} catch (err) {
```

Comment:
I don't think we should drop the `if (!purl.protocol)` check. Just move it here along with the call to `parse`, which isn't needed if its a local file.